src/views/ChannelView/ChannelView.tsx

```diff
@@ -71,6 +111,9 @@ const ChannelView: React.FC<RouteComponentProps> = () => {
               </>
             )}
           </TitleContainer>
+          <StyledButton variant={isFollowing ? 'secondary' : 'primary'} onClick={handleFollow}>
```

Comment:
Some issues here:

1. I know the design says otherwise but I think for the time being this should be centred vertically w.r.t. the channel avatar.
2. Take a look at [this example](https://deploy-preview-224--atlas-development.netlify.app/channel/0d75be46-9e81-4d79-a38f-753fbec0adf6) at `600px` wide screen. There are some issues here:
  2.1. The channel title doesn't collapse forcing the button outside the screen. This was introduced recently. You can fix that by adding `overflow: hidden;` to the `TitleContainer`
  2.2. Even with the previous point out of the way, there needs to be some padding between the title and the button.
3. Once you go below `600px` wide screen, there needs to be some spacing between the button and the followers count